tasks/app/initialize.py

```diff
@@ -22,19 +22,19 @@ def initialize_users_from_edm(context):
 
 
 @app_context_task
-def initialize_orgs_from_edm(context):
+def initialize_orgs_from_edm(context, edm_authentication=None):
```

Comment:
More a question than a comment, this edm_authentication is passed in but not apparently used, am I missing something??

Reply:
Yes, this is for automated testing in GitHub.